packages/cli/package.json

```diff
@@ -44,7 +44,7 @@
     "@celo/wallet-hsm-azure": "2.1.1-dev",
     "@celo/wallet-ledger": "2.1.1-dev",
     "@celo/wallet-local": "2.1.1-dev",
-    "@ledgerhq/hw-transport-node-hid": "~5.11.0",
+    "@ledgerhq/hw-transport-node-hid": "^6.27.4",
```

Comment:
I had a [PR to update ledger](https://github.com/celo-org/celo-monorepo/pull/9585/files) and i needed to do a few updates to code including one [slight esoteric change ](https://github.com/celo-org/celo-monorepo/pull/9585/files#diff-2289cdf2eeee340c9b6ce6b2d5d35f1b225994d5a22e8f5458b310a6caa48872L54-L59)